src/components/RenderConfig/RenderConfigComponent.tsx

```diff
@@ -306,7 +307,7 @@ export class RenderConfigComponent extends React.Component<WidgetProps> {
             percentileSelectDiv = (
                 <div className="percentile-select">
                     <FormGroup label="Clip Percentile" inline={true}>
-                        <HTMLSelect options={percentileRankOptions} value={frame.renderConfig.selectedPercentile} onChange={this.handlePercentileRankSelectChanged}/>
+                        <HTMLSelect options={percentileRankOptions} value={frame.renderConfig.selectedPercentile[stokes]} onChange={this.handlePercentileRankSelectChanged}/>
```

Comment:
suggestion: we should make a @computed value `selectedPercentileVal` similar to how you've handled the `scaleMinVal` computed values